pcsx2/DEV9/net.h

```diff
@@ -37,10 +37,31 @@ extern mtfifo<NetPacket*> rx_fifo;
 extern mtfifo<NetPacket*> tx_fifo;
 */
 
+enum struct NetApi : int
+{
+	Unset = 0,
+	PCAP_Bridged = 1,
+	PCAP_Switched = 2,
+	TAP = 3,
```

Comment:
Do we need to define the elements of the struct? I assume so if we save it as is on config but just asking in case

Reply:
The idea was that if we change anything in the enum (i.e. reorder, remove or add elements) we retain backwards compatibility with older config files (given we reference by int rather than name)
I could probably use the auto assigned enum values if we assume that won't be an issue.

I see we have ``IniInterface::EnumEntry()``, which may be the preferred way of storing enums if we ever port DEV9's config code to IniInterface (Out of scope for this PR however).

Reply:
It's fine, just wanted to ensure this wasn't an internal state we didn't store anywhere else, LGTM!